example_test.go

```diff
@@ -614,8 +614,8 @@ func ExampleDataset_WithCTERecursive() {
 	sql, _, _ := db.From("nums").
 		WithRecursive("nums(x)",
 			db.From().Select(goqu.L("1")).
-			UnionAll(db.From("nums").
-				Select(goqu.L("x+1")).Where(goqu.I("x").Lt(5)))).
+				UnionAll(db.From("nums").
+					Select(goqu.L("x+1")).Where(goqu.I("x").Lt(5)))).
```

Comment:
i think maybe go fmt would make these line not staggered.  I might be wrong but might wanna make sure you run go fmt against the project.

Reply:
Yeah it didnt reformat that section